san/backtest.py

```diff
@@ -22,40 +22,40 @@ def __init__(self, returns:pd.Series, trades:pd.Series, lagged=True, transaction
             trades.iloc[0] = False
         self.strategy_returns = ((returns * percent_invested_per_trade) * trades)
         self.trades = trades
-        
-        self.nr_trades = 0
+
+        self.nr_trades = {'buy': [], 'sell': []}
```

Comment:
`nr` was coming from `number`? if yes - this no longer keeps the number of trades, but list of trades

Reply:
yep
but variable trades already taken and nr_trades is mostly used to count number of sell/buy trades